bootstrap/sql/org.postgresql.Driver/v007__create_db_connection_info.sql

```diff
@@ -6,6 +6,6 @@ UPDATE data_insight_chart
 SET json = jsonb_set(
         json,
         '{dimensions}',
-        '[{"name":"entityFqn","chartDataType":"STRING"},{"name":"owner","chartDataType":"STRING"},{"name":"owner","entityType":"STRING"}]'
+        '[{"name":"entityFqn","chartDataType":"STRING"},{"name":"owner","chartDataType":"STRING"},{"name":"owner","entityType":"STRING"},{"name":"entityHref","entityType":"STRING"}]'
```

Comment:
Comparing this script with the MySQL one... I have 2 questions:
- Should this be `{"name":"entityHref","chartDataType":"STRING"}]`?
- Why do we have in this line 2 objects starting with `{"name":"owner"` but not in MySQL? I suspect that this line should be:
```suggestion
        '[{"name":"entityFqn","chartDataType":"STRING"},{"name":"owner","chartDataType":"STRING"},{"name":"entityType","chartDataType":"STRING"},{"name":"entityHref","chartDataType":"STRING"}]'
```
 in order to match the MySQL one.

Reply:
That is most likely from resolving conflict. The initial PR was merged by someone before I was able to push my updates from the comments on the PR